bmcs/src/test/resources/oci_api_key.pem

```diff
@@ -0,0 +1,30 @@
+-----BEGIN RSA PRIVATE KEY-----
+Proc-Type: 4,ENCRYPTED
+DEK-Info: AES-128-CBC,9D0B4B60F59A23522D7F0D8053EACB15
+
+YVU1WWqjGTPRSnoXK1Zz68GTfUYgZYDiJ30QeSNd+gSVC057dWv4/ONbLeIflCJ9
+6qQCdU0C5vMylV9Hu3Ej4aWPjXMOdi1GtSAtdMaUMlefU8KMI19szzV/Xn9JTCJN
+WxUg37XX+meoMTVolgplFY0dME4e5YDVfXumNtDs4JhRtODMRwBVT2jK8EvJttQ2
+06pJvb+qDU+4KWU4qydW+31DqqUqVg67HlvSHuf4QINoX77P7sZXJ0AdMgMlCcfn
++/KOJr0QsFPVEg6QZ5wSqFKq9k19Quco/igdYaNcYLTMd/xnN0butWCP0cxpARWO
+z/wisyrwOGU93uYLvUEUnzhOi8WHSh5HZ/riLNAlqP7I3+l6K1p9kSKfuPqB3oZp
+eCum9t9QO3pua9rdFmZ/eOBlu4oGgr4Grv4c54y/AeA78+21cifX+i3S/aRxTt/C
+9VsjVIEl+yzEd/kH063YOr35zw3YengIw+rfIa+gUpekWdlRfXJkFn/8bWzA90+M
+/rTWvdrlPqDK858cFOU1W2rPTq92+nHzd8rhGyFoDqoMHr8yPoEA51ZAczES63WW
+tMCjnm1SVDA7vDT1sHseTmzu3Rgi06/Y8m4gQvgAf5I9TEv73JI7JxynTvooen/P
+kQPnrGZeRLU0fUDpQUBzKczR7VXC6TZNof7dQeVfoVkQCKCZOfMFIemPhifTcIe5
+RyUd98VyOXM0fpO6QGShUCaZnGNzjYnQZgcK+zWSDvbqgbAB+352NGZJUPk5eIQ/
+izuFK9HxhYUC6xOZ7q1LVTuVaPQ5YQzY7rb0lfILLuOmBQS0VynVjfBMHILtjiVj
+2KQg6baQuqFDy1id1esmIk64VFLyJZX1TBZxEOpPhrbhy1Hs6ToE7m4ZG1/4oeRt
+Inc4k0IvYHE3eKxbUwYXZHhlS8PyqIqKsxmC6+wbVH4m3USJ6yA1DqP5S3gfSZJt
+JhPZFERJzAPPzePB19Ruby/NLX+sadTXUfcIVkhjThKUsZFkuTVCxkxgLLYUMQoY
+ASC9qHM9pqZz6g9ZEVUCgNRBSB4TZGH/dLHmuqJgQ+5/V7dStIaPJdv1oUJatn8l
+0dfPW64foXHu9HQJof9S1j8qAzXK0VJCv/RDtYRzI8cEY9sTf7EnAku1rwkKIozu
+ul4iIZnPlNzERuLBP7bq3u4QvnaIp71v41wmD9CMi61s7nHhr9G/lUCZB1JEKErQ
+KyxmjRMPIWTnkfHvM+/WNdpLYOb009vRG9Sn6pW7jNfduMcm/kr6eiVkxrfZvgvI
+Nipm4qbcT1eYKpbY/veSHtnXfQgv+28AnWiU+XTKqN+tCNuqT0hRGl4kBEk6nEw1
+BbhsgtAlGweVnKJywDi0ppJJL3ksyOfxaz3oA/VWoROSjzVSidFkxfb6fhOr+bZ5
+ALyG14gY2t1VK1SWFNFQtQxEqtQt4To8RTfHFaIN77tA8oT7+QMlo+Seg1fyAOcX
+DUsEU0RrWLImYGY3tzOLBLZFnSWUzZ9FDmUCa5lSHEKixpWTrw3jIvhOhNpecbsG
+6H+ichO/tjGy0vtzC2zLs/hF6tChvQbwZKkiEjwFFe2j4wUwRRE2KFQhw+ix2E14
+-----END RSA PRIVATE KEY-----
```

Comment:
This is just a dummy key, right?

Reply:
Yes. this was used for testing. no longer active.

Reply:
Please delete from the PR then